# Thomas Hoffmann, EMBL Heidelberg, structures-it@embl.de, 2023/09
# boost 0.1 @ pypi: 
# Boost automatically toots from users of the Mastodon social network
# ??? -> removed requirement
diff -ru PyTom-1.1/setup.py PyTom-1.1_fix_dependencies/setup.py
--- PyTom-1.1/setup.py	2023-08-29 17:09:10.000000000 +0200
+++ PyTom-1.1_fix_dependencies/setup.py	2023-09-01 12:26:05.552812907 +0200
@@ -62,7 +62,7 @@
     author='`FridoF',
     author_email='strubi.pytom@uu.nl',
     url='https://github.com/SBC-Utrecht/PyTom.git',
-    install_requires=['lxml', 'scipy', 'boost', 'numpy'],
+    install_requires=['lxml', 'scipy', 'numpy'],
     extras_require={
         'gpu': ['cupy'],
         'gui': ['PyQt5', 'pyqtgraph', 'mrcfile'],
